CHANGELOG.md

```diff
@@ -8,6 +8,24 @@ Added
 -
 
 * Add HPA on ArmoniK control plane based on CPU and RAM utilization metrics
+* Add node selector on all deployments (ArmoniK and Helm charts for infrastructure)
+* Add logs of metrics exporter, MongoDB, NGINX, and Keda in CloudWatch via Fluent-bit
+* Add code formatter
+* Add PayloadId to database
+
+Changed
+-
+
+* Change default branch to main instead master
```

Comment:
As it is an implementation details of ArmoniK.Core, I don't think it should appear in this Changelog.